test_files/variant_pretty.slt

```diff
@@ -0,0 +1,21 @@
+statement ok
```

Comment:
this is amazing -- thank you

It makes me wonder if we should have some way to override the default "displayable" format for types in DataFusion (aka so we don't need a variant pretty function)